Dockerfile

```diff
@@ -60,7 +60,9 @@ LABEL description="PMEM CSI Driver"
 # storge-utils - for lvm2 and ext4(e2fsprogs) utilities
 ARG CACHEBUST
 RUN echo "Updating runtime image from ${CLEAR_LINUX_BASE} to ${SWUPD_UPDATE_ARG:-the latest release}."
-RUN swupd update ${SWUPD_UPDATE_ARG} && swupd bundle-add file xfsprogs storage-utils && rm -rf /var/lib/swupd /var/tmp/swupd
+RUN swupd update ${SWUPD_UPDATE_ARG} && swupd bundle-add file xfsprogs storage-utils \
+    $(if [ "$BIN_SUFFIX" = "-test" ]; then echo fio; fi) && \
```

Comment:
Doesn't quite work yet because `BIN_SUFFIX` is not an argument in this section of the `Dockerfile`.

Here's the fix:
```
diff --git a/Dockerfile b/Dockerfile
index ae8fac19..159285ad 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -51,6 +51,7 @@ RUN echo "For source code and licensing of ndctl, see https://github.com/pmem/nd
 FROM ${CLEAR_LINUX_BASE} as runtime
 ARG CLEAR_LINUX_BASE
 ARG SWUPD_UPDATE_ARG
+ARG BIN_SUFFIX
 LABEL maintainers="Intel"
 LABEL description="PMEM CSI Driver"
```

With that it worked for me:
```
$ docker run -ti --rm localhost:5000/pmem-csi-driver-test:canary /bin/bash
root@d64d7f799034/ # ls /usr/local/bin/
pmem-csi-driver  pmem-dax-check  pmem-ns-init  pmem-vgm
root@d64d7f799034/ # fio --help
fio-3.19
fio [options] [job options] <job file(s)>
  --debug=options	Enable debug logging. May be one/more of:
			process,file,io,mem,blktrace,verify,random,parse,
...
```

Reply:
And it also worked as intended for the non-test image.

Reply:
I added ARG like proposed